packages/protocol-contracts/contracts/ZetaConnector.non-eth.sol

```diff
@@ -101,4 +104,8 @@ contract ZetaConnectorNonEth is ZetaConnectorBase {
             internalSendHash
         );
     }
+
+    function setMaxSupply(uint256 maxSupply) external whenNotPaused onlyTssAddress {
```

Comment:
I don't think `whenNotPaused` is a feature here, and I can think about the case of us pausing then setting the maxSupply, so I'd remove that modifier.

Reply:
hmm... I think perhaps additionally the admin (TSSAddressUpdater?) should be able to change this parameter
and behind the same timelock contract